mbofui/Gruntfile.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(grunt) {
       options: {
         banner: '/*\n <%= pkg.name %> <%= pkg.version %> <%= grunt.template.today("yyyy-mm-dd") %> \n*/\n',
         sourceMap: true,
-        sourceMapName: 'all.min.js.map',
+        sourceMapName: 'app/js/dist/all.min.js.map',
```

Comment:
I was writing out the file to the root of the project. It needs to be in the same space as the concatenated js. I was seeing errors in the logs because of this - they are no longer.